Update the game over message (#1)
elsaahmed1/skills-review-pull-requests: README.md

</header>

<!--
  <<< Author notes: Step 5 >>>
 Just editing this step by acknowledging the previous step.
  Define terms and link to docs.github.com.
-->

> ## Step 5: Apply suggested changes

Author: ```suggestion
## Step 5: Apply suggested changes
```

_Nicely done suggesting changes! :partying_face:_

Now let's see how easy it is to [apply your suggestion](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/incorporating-feedback-in-your-pull-request).

> ### :keyboard: Activity: Apply suggested changes

Author: ```suggestion
### :keyboard: Activity: Apply suggested changes here
```

</header>

<!--
  <<< Author notes: Step 4 >>>

>   Start this step by acknowledging the previous step.

Author: ```suggestion
 Just editing this step by acknowledging the previous step.
```
